admin/class-monk-admin.php

```diff
@@ -1449,24 +1449,75 @@ public function monk_change_nav_menu_fields() {
 	 *
 	 * @since    0.4.0
 	 *
-	 * @return void
+	 * @return array
 	 */
 	public function monk_save_language_packages() {
-		if ( check_ajax_referer( '_monk_nonce', '_monk_nonce', false ) ) {
-			$active_languages = $_POST['monk_active_languages'];
+		$active_languages = $_POST['monk_active_languages'];
```

Comment:
@leonofre accordingly to `filter_input` documentation:
> FILTER_DEFAULT (integer)
ID do filtro padrão ("string").

So you'll need specify another filter to use `filter_input` here.
http://php.net/manual/pt_BR/filter.constants.php

I would start trying by `FILTER_REQUIRE_ARRAY`